From 1a3d3784a08d84892c60427b8f7be1cea7d38411 Mon Sep 17 00:00:00 2001
From: Mateusz Stadnik <matgla@live.com>
Date: Tue, 23 Jul 2024 22:18:07 +0200
Subject: [PATCH] Optimized pio_blink for renode

---
 pio/pio_blink/blink.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pio/pio_blink/blink.c b/pio/pio_blink/blink.c
index 9d253bb..26fd686 100644
--- a/pio/pio_blink/blink.c
+++ b/pio/pio_blink/blink.c
@@ -24,6 +24,9 @@ int main() {
     blink_pin_forever(pio, 0, offset, 0, 3);
     blink_pin_forever(pio, 1, offset, 6, 4);
     blink_pin_forever(pio, 2, offset, 11, 1);
+
+    // otherwise it execute infinite loop in _exit which slows down simulation extremaly 
+    while (true) { sleep_ms(5000); }
 }
 
 void blink_pin_forever(PIO pio, uint sm, uint offset, uint pin, uint freq) {
-- 
2.44.0

