From 0a100bf7a3f715f1ac6dc7118dba453567939635 Mon Sep 17 00:00:00 2001
From: Mateusz Stadnik <matgla@live.com>
Date: Mon, 28 Oct 2024 15:52:17 +0100
Subject: [PATCH] added newlines in hello multicore

---
 multicore/hello_multicore/multicore.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/multicore/hello_multicore/multicore.c b/multicore/hello_multicore/multicore.c
index 7ad036c..e10384b 100644
--- a/multicore/hello_multicore/multicore.c
+++ b/multicore/hello_multicore/multicore.c
@@ -19,7 +19,7 @@ void core1_entry() {
     if (g != FLAG_VALUE)
         printf("Hmm, that's not right on core 1!\n");
     else
-        printf("Its all gone well on core 1!");
+        printf("Its all gone well on core 1!\n");
 
     while (1)
         tight_loop_contents();
@@ -41,7 +41,7 @@ int main() {
         printf("Hmm, that's not right on core 0!\n");
     else {
         multicore_fifo_push_blocking(FLAG_VALUE);
-        printf("It's all gone well on core 0!");
+        printf("It's all gone well on core 0!\n");
     }
 
     /// \end::setup_multicore[]
-- 
2.44.0

